packages/medusa/src/api/routes/admin/payment-collections/delete-payment-collection.ts

```diff
@@ -0,0 +1,13 @@
+import { PaymentCollectionService } from "../../../../services"
+
+export default async (req, res) => {
+  const { id } = req.params
+
+  const paymentCollectionService: PaymentCollectionService = req.scope.resolve(
+    "paymentCollectionService"
+  )
+
+  const payment_collection = await paymentCollectionService.delete(id)
+
+  res.status(200).json({ payment_collection })
```

Comment:
todo: Should follow general convention for `DELETE`s
```suggestion
  res.status(200).json({ id, deleted: true, object: "payment_collection" })
```